<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeManagementWindow</class>
 <widget class="QWidget" name="EmployeeManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento de Jogos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #D7D7D7;
    }
    QLabel {
        color: #2D4B50;
        font-family: &quot;Arial Black&quot;;
        margin: 5px;
        min-width: 300px; /* Isso pode não funcionar como esperado para QLabel no styleSheet */
    }
    QPushButton, QLineEdit, QDateEdit, QComboBox {
        color: #2D4B50;
        font-family: &quot;Arial Black&quot;;
        margin: 5px;
    }
    QLineEdit, QDateEdit, QComboBox {
        background-color: #fff;
        border: 1px solid #555;
        border-radius: 5px;
        padding: 1px 5px;
        height: 25px;
    }
    QPushButton {
        background-color: #2D4B50;
        color: white;
        border-radius: 5px;
        padding: 5px;
        margin: 10px;
        height: 40px;
    }
    QPushButton:hover {
        background-color: #6C6C69;
    }
    QTableWidget, QTableWidget QHeaderView::section, QTableWidget::item {
        background-color: #fff;
        color: #2D4B50;
        border: 1px solid #555;
        border-radius: 5px;
    }
    QTableWidget QHeaderView::section {
        background-color: #E8E8E8; /* Um pouco mais claro que #D7D7D7 para contraste */
        color: #2D4B50;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../assets/logo.qrc">:/res/logo.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome Completo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>E-mail</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CPF</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonRemoveEmployee">
       <property name="text">
        <string>Remover Funcionário</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonModifyEmployee">
       <property name="text">
        <string>Alterar Funcionário</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../assets/logo.qrc"/>
 </resources>
 <connections/>
</ui>
